thesslstore_module.php

```diff
@@ -3719,8 +3800,86 @@ private function parseResponse($response, $ignore_error = false) {
         return $response;
     }
 
+    /**
+     * Returns all fields to display to an admin attempting to edit a service with the module
+     *
+     * @param stdClass $package A stdClass object representing the selected package
+     * @param $vars stdClass A stdClass object representing a set of post fields
+     * @return ModuleFields A ModuleFields object, containg the fields to render as
+     *  well as any additional HTML markup to include
+     */
+    public function getAdminEditFields($package, $vars = null)
+    {
+        Loader::loadHelpers($this, ['Html']);
 
+        $fields = new ModuleFields();
 
+        // Create domain label
```

Comment:
I know the rest of this file does not adhere to it, but we try to have a maximum line length of 120 characters.